nucliadb_node/src/shards/scheduler.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
+//
+// nucliadb is offered under the AGPL v3.0 and as commercial software.
+// For commercial licensing, contact us at info@nuclia.com.
+//
+// AGPL:
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+use std::sync::Arc;
+
+use nucliadb_core::tracing::error;
+use nucliadb_core::NodeResult;
+use tokio::sync::{Semaphore, SemaphorePermit};
+use tokio::time::{self, Duration, Interval};
+
+use crate::shards::shard_writer::ShardWriter;
+
+// Every three hours a merge is triggered
+const MERGE_INTERVAL_SECS: u64 = 10800;
+// Everyday a shard is garbage collected
+const GC_INTERVAL_SECS: u64 = 10800;
```

Comment:
that's also 3 hours in seconds, and your comment says every day. Tipo?

Reply:
Yes tipo.. sorry about that!